apps/src/aiTutor/redux/aiTutorRedux.ts

```diff
@@ -105,7 +104,7 @@ export const askAITutor = createAsyncThunk(
     if (chatApiResponse.assistantResponse) {
       const assistantChatMessage: ChatCompletionMessage = {
         role: Role.ASSISTANT,
-        status: Status.OK,
+        status: chatApiResponse.status,
```

Comment:
Storing the `chatApiResponse` status from the server lets us style the messages appropriately later.